mocks/dummyData.ts

```diff
@@ -849,6 +849,242 @@ export const auth: Authorization = {
   },
 }
 
+export const auth2 = {
```

Comment:
Glad you added this, we had to do a lot of work around the old auth in dummy data